docs/howtos/integrations/integration-context.md

```diff
@@ -0,0 +1,73 @@
+---
+id: integration-context
+title: Integration Cache
+---
+
+## Overview
+
+Occasionally, you might need to store data between integration commands runs.
+
+A common use-case would be storing API tokens which have expiration time (i.e. JWT).
+
+For that, Demisto introduces the cached object `integrationContext`.
+
+The object is stored in the database per integration instance.
+
+## Implementation
+
+The `integrationContext` supports two methods: getter and setter.
+
+If no object is stored, the method will return an empty dictionary.
+
+### The `getIntegrationContext()` Method
+This is the getter the cached object, which returns a key-value dictionary.
+
+### The `setIntegrationContext()` Method
+This is the setter the cached object.
+
+This method takes only one argument - the object to store, which its keys and values must be strings.
+
+Note that this method overrides the existing object which is stored, so in order to update a stored object, one should first get it, make the requested changes and then set it.
+
+
+## Examples
+
+### General usage
+```python
+integration_context: Dict = demisto.getIntegrationContext()
+print(integration_context)
```

Comment:
`print` is usually not used in integrations (at least we don't advocate for it to be used in other documents`. Can we replace it with `demisto.log()` or `demisto.INFO()` ?